hw/ip/hmac/data/hmac.hjson

```diff
@@ -468,7 +470,7 @@
               For SHA-2 384, {DIGEST12-DIGEST15} are truncated; they are irrelevant and should not be read out.
 
               The digest gets cleared when `CFG.sha_en` transitions from 1 to 0.
-              When `CFG.sha_en` is 0, these registers can be written to by software.
+              When `STATUS.hmac_idle` is 1, these registers can be written to by software. Outside of this window, writes can cause unpredictable results.
```

Comment:
Slight rephrase but otherwise LGTM (also for the other two changes below)
```suggestion
              When `STATUS.hmac_idle` is 1, these registers may be written to by software. Outside of this window, writes can cause unpredictable behavior.
```

Reply:
And please use 1 sentence per line.

Reply:
Thanks for your comments, all are in my latest commit.